### DISCUSSION TITLE
VHD translation IT

### DISCUSSION THREAD

**Discussion by Fritzontheway** [aclap-dev/video-downloadhelper#1198] [Language & Translation]
Hi, here attached the italian translation of missing lines

**Comment by jcfrog** [comment]
Hi,
Thanks but I think you forgot to attach the file.

**Comment by Temistocle** [comment]
[20240901 vdh it.txt](https://github.com/user-attachments/files/16828480/20240901.vdh.it.txt)
Hi, I'm enclosing my translation contribution. 
Kindest regards, 
giacomo

**Comment by Fritzontheway** [comment]
Hi, tried but clicking on the “attach files” icon nothing happen :(

Here my file.

All the best

 

Da: jcfrog ***@***.*** 
Inviato: domenica 1 settembre 2024 09:52
A: aclap-dev/video-downloadhelper
Cc: Fritzontheway; Author
Oggetto: Re: [aclap-dev/video-downloadhelper] VHD translation IT (Discussion #1198)

 

Hi,
Thanks but I think you forgot to attach the file.

—
Reply to this email directly, view it on GitHub <https://github.com/aclap-dev/video-downloadhelper/discussions/1198#discussioncomment-10510739> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/BCGIE7SEMYF5RA7WVERANKTZULBR3AVCNFSM6AAAAABNOAKGHOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJRGA3TGOI> .
You are receiving this because you authored the thread.Immagine rimossa dal mittente.Message ID: ***@***.***>


[it]
# v9.1.0.42
v9_panel_footer_show_history_tooltip = '''Mostra la cronologia dei download'''
v9_history_page_title = '''La tua cronologia dei download'''
v9_history_button_clear = '''Pulisci la cronologia'''
v9_history_button_start_recording = '''Ricorda la cronologia'''
v9_history_button_stop_recording = '''Non ricordare la cronologia'''
v9_history_no_entries = '''Nessuna'''
v9_history_input_search = '''Cerca'''
v9_settings_history_limit = '''Dimentica la cronologia dopo X giorni'''
v9_date_today = '''oggi'''
v9_date_yesterday = '''ieri'''
v9_date_x_days_ago = '''$1 giorni fa'''
v9_date_long_ago = '''molto tempo fa'''
v9_history_no_recording_description_safe = '''Non preoccuparti, tutto rimane sul tuo computer. La tua privacy è importante per noi.'''
v9_history_no_recording_description = '''Non registriamo la cronologia dei tuoi download. Vuoi che Video DownloadHelper memorizzi la cronologia dei tuoi download?'''
